charts/perses/values.yaml

```diff
@@ -178,6 +178,8 @@ sidecar:
   label: "perses.dev/resource"
   # -- Label value to watch for ConfigMaps containing Perses resources
   labelValue: "true"
+  # -- check for configmaps from all namespaces. When set to false, it will only check for configmaps in the same namespace as the Perses instance
+  allNamespaces: true
```

Comment:
Sorry for the late question.
Do you think that we should make it false by default, so we can ensure the previous behaviour.

This is a open ending question?

Reply:
I integrated [this comment](https://github.com/perses/helm-charts/blob/1c1b7674d9fe61ba456791a1e0888d3ce905bf18/charts/perses/values.yaml#L168) that this sidecar container checks for configmaps in all the namespaces. So, I considered this PR as a bug fix 😅. 
However, I don't have any strong opinion on this. If you think, making it false by default makes more sense then I will be happy to do it :)

Reply:
I don't have it too, so I'm fine with this as default true.